python/hsfs/util.py

```diff
@@ -220,6 +221,34 @@ def get_hostname_replaced_url(sub_path: str):
     return url_parsed.geturl()
 
 
+def verify_attribute_key_names(feature_group_obj):
+    feature_names = set([feat.name for feat in feature_group_obj.features])
```

Comment:
```suggestion
    feature_names = set(feat.name for feat in feature_group_obj.features)
```
unnecessary to convert between the two